gito/commands/deploy.py

```
@@ -0,0 +1,98 @@
+        "Do you want to commit and push created GitHub workflows to a new branch?"
+    ):
+        repo.git.add([str(file) for file in workflow_files.values()])
+        branch_name = "gito_deploy"
+        if not repo.active_branch.name.startswith(branch_name):
+            repo.git.checkout("-b", branch_name)
+        repo.git.commit("-m", "Deploy Gito workflows")
+        repo.git.push("origin", branch_name)
+        print(f"Changes pushed to {branch_name} branch.")
+        print(
+            f"Please create a PR from {branch_name} to your main branch and merge it:\n"
+            f"https://github.com/{owner}/{repo_name}/compare/gito_deploy?expand=1"
```

Flake8 (7.2.0)

[error] undefined name 'owner'

(F821)